config.yml

```
@@ -1,5 +1,6 @@
 default:
   parent: "syn20400157"
+  path_to_markdown: !expr system.file("using-the-dccvalidator-app-amp-ad.Rmd", package = "dccvalidator")
```

```suggestion
    path_to_markdown: "inst/using-the-dccvalidator-app-amp-ad.Rmd"
```
I know I suggested this, but it is not working when I run the app from my fork. After I do `load_all()`, `system.file("using-the-dccvalidator-app-amp-ad.Rmd", package = "dccvalidator")` points to the local copy of the Rmd, not the installed one, which is fine but for some reason `config::get("path_to_markdown")` only returns an empty string. 

I suggest that for our copy of the config file, we use the local copy of the Rmd. If someone installs the package without forking and wants to use our documentation, they could still do that with `system.file()`. They wouldn't run into the issue above since they wouldn't be using `load_all()`.

---

At what point does a user execute `load_all()`? I only familiar with deploying the app using the [instructions](https://sage-bionetworks.github.io/dccvalidator/articles/deploying-dccvalidator.html).

---

It's in the `app.R` file, so it gets run if you deploy the app by forking & cloning the repo

---

Ok, got it!